deploy/script/katello-restore

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# Echo all commands
+usage(){
+  echo "Usage: katello-restore <backup directory>"
+  echo " eg: $ katello-restore /tmp/katello-backup-201501261543"
```

Comment:
This is still suggesting something different than what the backup script suggests  and it would be nice if the coincided.